apps/supabase-auth/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Hono } from 'hono';
+import { createClient } from '@supabase/supabase-js';
+
+const app = new Hono();
+
+app.get('/me/:access_token', async (c) => {
+  const supabasePublicKey = 'placeholder';
```

Comment:
Instead of hardcoding these here use env vars (see https://hono.dev/getting-started/cloudflare-workers#bindings), then define your env vars in .dev.vars file